app/src/main/java/chat/rocket/android/layouthelper/chatroom/AbstractMessageViewHolder.java

```diff
@@ -14,6 +14,7 @@
 public abstract class AbstractMessageViewHolder extends ModelViewHolder<PairedMessage> {
   protected final RocketChatAvatar avatar;
   protected final ImageView userNotFoundAvatarImageView;
+  protected final ImageView imageViewSvgUserAvatar;
```

Comment:
We need, because `RocketChatAvatar` is a class which returns a [SimpleDraweeView](http://frescolib.org/docs/using-simpledraweeview.html). There is a `setImageDrawable()` method on SimpleDraweeView but it is deprecated (trying to use it you'll get the messed avatars). Also, from [Fresco](http://frescolib.org/docs/gotchas.html):

> **Don’t set images directly on a DraweeView**
Currently DraweeView is a subclass of Android’s ImageView. This has various methods to set an image (such as setImageBitmap, setImageDrawable)
If you set an image directly, you will completely lose your DraweeHierarchy, and will not get any results from the image pipeline.